src/include/sof/audio/generic_processor/hifi_codec/xa_type_def.h

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
I think we should introduce Cadence provided interface files at the beginning in a separate file, so that we can have all of them in one commit. What do you think?

Reply:
hmm why would we do so? This would create a bit *dummy* commits without any functionality behind them. I think the idea is to make a clear distinction between Cadence provided code and ours, I sort of buy this idea but in the same time I don't see it as a necessary step.